app/javascript/controllers/litigation_side_controller.js

```diff
@@ -0,0 +1,42 @@
+import { Controller } from 'stimulus';
+
+const ENTITY_TYPE_PARTY_TYPE_MAP = {
+  'Company': 'corporation',
+  'Location': 'government'
+};
+
+export default class extends Controller {
+  connect() {
+    $(this.connectedWithSelect).on('change', this._setDefaults.bind(this));
+  }
+
+  _setDefaults() {
+    const selectedOption = this.connectedWithSelect.selectedOptions &&
```

Comment:
I'd extract `this.getSelectedOption()` for this.

Reply:
:+1: